etl/src/main/java/org/pih/hivmigration/etl/sql/ExamLabResultsMigrator.groovy

```diff
@@ -113,7 +113,7 @@ class ExamLabResultsMigrator extends ObsMigrator {
         ''')
 
         executeMysql("Log warning about invalid PPD values", '''
-            INSERT INTO hivmigration_data_warnings (patient_id, encounter_id, field_name, field_value, note)
+            INSERT INTO hivmigration_data_warnings (openmrs_patient_id, openmrs_encounter_id, field_name, field_value, warning_type)
```

Comment:
iirc, the note was something like generic like "Invalid valid."  I think we'd want something like "Invalid PPD value", so our aggregrate counts (which group by "warning_type") would be by test.  You could check with Debbie and/or Ricard and see what they prefer, but I assume they will want to tackle the DQ issues in the more important tests first, so a count by test would make sense.

Reply:
Yes, good call. I'll update these.

Reply:
https://github.com/PIH/hivmigration/pull/69